src/parallel.cpp

```diff
@@ -471,12 +468,12 @@ void CParallelValidation::HandleBlockMessage(CNode *pfrom,
             } // We must not hold the lock here because we could be waiting for a grant, below.
 
             // wait for semaphore grant
-            semPV->wait();
+            semaphore.wait();
```

Comment:
I agree, a semaphore's name should reference what resource its controlling access to.